tsconfig.json

```diff
@@ -14,12 +17,28 @@
     "jsxFactory": "h",
     "jsxFragmentFactory": "Fragment",
     "paths": {
-      "nitropack": ["./src/index"],
-      "nitropack/config": ["./src/config"],
-      "#internal/nitro": ["./src/runtime/index"],
-      "#internal/nitro/*": ["./src/runtime/*"]
+      "nitropack": [
+        "./src/index"
```

Comment:
Please consider moving unrelated changes like this + playground types to separate PRs to avoid blocking merges and allow more clear changelogs 🙏🏼